shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
Yep, you need to modify your `16.04-server.s390` to `16.04-server.s390x`. Basically take a look at the `Fedora.23` definition. Anyway I'd be even fine with s390, my problem is only with having the machine in `machines.cfg` and the arch the same.

Btw the cartesian_config is simply clever, it knows nothing about files, nor locations. It creates a global namespace of terms and filters them. So `only s390` basically looks for any occurrence of `s390` in the global namespace. So if you define:

```
s390:
    only s390
```
it means always include it, because you're in s390 branch. That's why picking a good unique name is very important in cartesian, everything is global.